server/routes/addPrisoner/prisonerLocationController.test.ts

```diff
@@ -72,6 +74,25 @@ describe('Prisoner location', () => {
         })
     })
 
+    it('should pre-populate prison choice if prison ID set in session', () => {
+      sessionData.addPrisonerJourney = { supportedPrisonIds: undefined, selectedPrisonId: 'HEI' }
+
+      return request(app)
+        .get(paths.ADD_PRISONER.LOCATION)
+        .expect('Content-Type', /html/)
+        .expect(res => {
+          const $ = cheerio.load(res.text)
+          expect($('input[name=prisonId]').length).toBe(1)
+          expect($('input[name=prisonId]:checked').length).toBe(1)
+          expect($('input[name=prisonId]').eq(0).val()).toBe('HEI')
+
+          expect(sessionData.addPrisonerJourney).toStrictEqual<AddPrisonerJourney>({
```

Comment:
Realised you can use generics (`<AddPrisonerJourney>`) to type the `toStrictEqual()` call which is quite handy.